src/components/organisms/AdminVenueView/components/Spaces/Spaces.scss

```diff
@@ -0,0 +1,173 @@
+@import "scss/constants.scss";
+
+.Spaces {
+  display: flex;
+  flex: 1;
+
+  &__rooms {
+    width: 20%;
+  }
+
+  &__map {
+    width: 80%;
+    background-color: #19181a;
+    justify-content: center;
+    align-items: center;
+    padding: 40px;
+  }
+
+  &__background {
+    padding: 20px;
+  }
+
+  &__upload-background-wrapper {
+    background-color: #19181a;
+    border-radius: 16px;
+    padding: 40px;
+  }
+
+  &__upload-background-button {
+    background-color: rgba(255, 255, 255, 0.2);
+    border-radius: 26px;
+    padding: 10px;
+    text-align: center;
+  }
+
+  &__backgrounds {
+    display: flex;
+    position: relative;
+    overflow-x: scroll;
+  }
+
+  &__title {
+    font-size: 2rem;
+    font-weight: bolder;
+  }
```

Comment:
Please, use our consts everywhere

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553